### DISCUSSION TITLE
Update "Support site" information on the Chrome store

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#3321] [General]
@gorhill 

Open the AdGuard download page and scroll down to the bottom of the page: https://chromewebstore.google.com/detail/adguard-adblocker/bgnkhhnnamicmpeenaelnjfhikgbkllg . It point users to the appropriate support site:

>Support
>For help with questions, suggestions, or problems, visit the developer's [support site](https://github.com/AdguardTeam/AdguardBrowserExtension/issues)

Your extensions point to Chrome Web Store's internal support hub instead, which isn't monitored by anybody:

https://chromewebstore.google.com/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm/support
https://chromewebstore.google.com/detail/ddkjiahejlhfcafbddmgiahcphecmpfh/support

**Comment by gorhill** [comment]
I did provide a "Support URL" link to `https://github.com/gorhill/uBlock` since forever, but somehow the Chrome Web Store is not using it. I just changed it to `https://github.com/uBlockOrigin/uBlock-issues` to see if this will make a difference.
